### PR TITLE
Initial documentation on PGP signatures verification

### PR DESCRIPTION
**Pull request by nijel**

### CODE REVIEW

**Comment by lem9** [comment]
See https://github.com/nijel/phpmyadmin/pull/1

File: doc/setup.rst [hunk 1]
```diff
@@ -245,6 +246,82 @@ options which the setup script does not provide.
...
+    gpg:          There is no indication that the signature belongs to the owner.
+    Primary key fingerprint: 436F F188 4B1A 0C3F DCBF  0D79 FEFC 65D1 81AF 644A
+
+The problem here is that anybody could issue the key with this name.  You need to
+ensure that the key is actually owned by the mentioned person.  The GNU Privacy
+Handbook covers this topic in the chapter `Validating other keys on your public
+keyring`_. The most reliable method is to meet the developer in person and
+exchange key fingerprints, however you can also rely on the web of trust. This way
+you can trust the key transitively though signatures of others, who have met
+the developer in person. For example you can see how `Marc's key links to Linus
```
Comment (ibennetch) [inline doc/setup.rst]: "Marc's key links to Linus's key" or "Marc's key links to Linus's" is probably better than "Linus one" here

File: doc/setup.rst [hunk 2]
```diff
@@ -245,6 +246,82 @@ options which the setup script does not provide.
...
+one`_.
+
+Once the key is trusted, you should get rid of that warning:
```
Comment (ibennetch) [inline doc/setup.rst]: - you should get rid of that warning
+ the warning should not be shown
or
+ the warning will not occur

**Comment by nijel** [comment]
Thanks Isaac for suggestions, I've applied them. Any other comments?

File: doc/setup.rst [hunk 3]
```diff
@@ -245,6 +246,82 @@ options which the setup script does not provide.
...
+
+.. code-block:: console
+
+    $ gpg --verify phpMyAdmin-4.4.9-all-languages.zip.asc
+    gpg: Signature made Fri Jun 12 13:09:58 2015 CEST using RSA key ID 81AF644A
+    gpg: BAD signature from "Marc Delisle <marc@infomarc.info>" [unknown]
+
+.. _Validating other keys on your public keyring: https://www.gnupg.org/gph/en/manual.html#AEN335
+
+.. _Marc's key links to Linus one: http://pgp.cs.uu.nl/mk_path.cgi?FROM=00411886&TO=81AF644A
```
Comment (ibennetch) [inline doc/setup.rst]: This text should also be updated to match line 300-301.

**Comment by ibennetch** [comment]
I made one more suggestion but other than that it looks good to me. Nice work.

**Pull request merged**